sdk/textanalytics/ai-text-analytics/src/analyzeSentimentResultResponse.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as coreHttp from "@azure/core-http";
+import {
+    GeneratedClientSentimentResponse,
+    SentimentResponse,
+    TextDocumentInput
+  } from "./generated/models";
+
+import { AnalyzeSentimentResultArray, makeAnalyzeSentimentResultArray } from './analyzeSentimentResultArray';
+
+/**
+ * Contains response data for the Analyze Sentiment operation.
+ */
+export type AnalyzeSentimentResultResponse = AnalyzeSentimentResultArray & {
+    /**
+     * The underlying HTTP response.
+     */
+    _response: coreHttp.HttpResponse & {
+        /**
+         * The response body as text (string format)
+         */
+        bodyAsText: string;
+    
+        /**
+         * The response body as parsed JSON or XML
+         */
+        parsedBody: SentimentResponse;
+      };
+  };
+
+export function toAnalyzeSentimentResultResponse(
+    input: TextDocumentInput[],
+    response: GeneratedClientSentimentResponse
+): AnalyzeSentimentResultResponse {
+    return Object.assign(makeAnalyzeSentimentResultArray(input, response), {
```

Comment:
usually `_response` is non-enumerable, in part because headers can contain secret information and also folks don't expect to see it if they JSON.stringify() the object. I think you want to use `Object.defineProperty` here instead?

Reply:
Done.